pyroute/run.py

```diff
@@ -1,3 +1,23 @@
+""" Run module
+
+    The module 'Run' is the piece of code that is in charge of the
+
+    set up of the configuration and the tests of the scripts the user
+
+    wants to be tested.
+
```

Comment:
@marioBZ7 can you remove the blank space between each line, from line 4 to line 7?